tests/Tests/200__monitor_and_manage/210__alerts/210__alerts.robot

```diff
@@ -204,6 +206,76 @@ Verify Alert "RHODS Probe Success Burn Rate" Is Fired When RHODS Dashboard Is Do
 
     [Teardown]    ODS.Restore Default Deployment Sizes
 
+Verify Alert "JupyterHub image builds are failing" Fires When There Is An Image Build Error
+    [Documentation]     Verify the "JupyterHub image builds are failing" alert when there is a image build failed
+    [Tags]    Sanity
+    ...       Tier2
+    ...       ODS-717
+    ...       KnownIssues
+    ${failed_build_name} =    Provoke Tensorflow Image Build Failure
+    Prometheus.Wait Until Alert Is Firing    ${RHODS_PROMETHEUS_URL}
+    ...    ${RHODS_PROMETHEUS_TOKEN}
+    ...    Builds
+    ...    JupyterHub image builds are failing
+    ${build_name} =    Start New Build    namespace=redhat-ods-applications
+    ...    buildconfig=s2i-tensorflow-gpu-cuda-11.4.2-notebook
+    Prometheus.Wait Until Alert Is Not Firing    ${RHODS_PROMETHEUS_URL}
+    ...    ${RHODS_PROMETHEUS_TOKEN}
+    ...    Builds
+    ...    JupyterHub image builds are failing
+    Wait Until Build Status Is    namespace=redhat-ods-applications    build_name=${build_name}
+    Prometheus.Alert Should Not Be Firing    ${RHODS_PROMETHEUS_URL}
+    ...    ${RHODS_PROMETHEUS_TOKEN}
+    ...    Builds
+    ...    JupyterHub image builds are failing
+    Sleep  10m   reason=Waiting for the alert to keep not firing
+    Prometheus.Alert Should Not Be Firing    ${RHODS_PROMETHEUS_URL}
+    ...    ${RHODS_PROMETHEUS_TOKEN}
+    ...    Builds
+    ...    JupyterHub image builds are failing
+    Sleep  10m   reason=Waiting for the alert to keep not firing
+    Prometheus.Alert Should Not Be Firing    ${RHODS_PROMETHEUS_URL}
+    ...    ${RHODS_PROMETHEUS_TOKEN}
+    ...    Builds
+    ...    JupyterHub image builds are failing
+    Sleep  10m   reason=Waiting for the alert to keep not firing
+    Prometheus.Alert Should Not Be Firing    ${RHODS_PROMETHEUS_URL}
+    ...    ${RHODS_PROMETHEUS_TOKEN}
+    ...    Builds
+    ...    JupyterHub image builds are failing
+    [Teardown]    Delete Build    namespace=redhat-ods-applications    build_name=${failed_build_name}
+
+Verify Alert "JupyterHub Image Builds Are Failing" Fires Up To 30 Minutes When There Is An Image Build Error
+    [Documentation]     Verify that alert is firing up to 30 min and after thar resolve automatically
+    [Tags]    Sanity
+    ...       Tier2
+    ...       ODS-790
+    ...       KnownIssues
+    ${failed_build_name} =    Provoke Tensorflow Image Build Failure
```

Comment:
@gmfrasca said that a way not to have the two build alert tests interfere each other could be to use tensorflow build in one test and pytorch build in the other

Could you add the build name as parameter and use different builds on each test?

```
Provoke Image Build Failure
[Arguments]    ${build_name_includes}
```

Reply:
Done